### PR TITLE
Rebrand table indexes

### PR DESCRIPTION
**Pull request by matteotrubini**
Ensure coherence in schema definitions

### CODE REVIEW

File: updates/v2.0.1/rename_indexes.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            $this->updateIndexNames($from, $to, $to);
+        }
+    }
+
+    public function down()
+    {
+        foreach (self::TABLES as $table) {
+            $from = 'winter_translate_' . $table;
+            $to   = 'rainlab_translate_' . $table;
+            $this->updateIndexNames($from, $to, $from);
```
Comment (mjauvin) [inline updates/v2.0.1/rename_indexes.php:29]:
```suggestion
            $this->updateIndexNames($from, $to);
```

File: updates/v2.0.1/rename_indexes.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    }
+    
+    public function updateIndexNames($from, $to, $table)
```
Comment (mjauvin) [inline updates/v2.0.1/rename_indexes.php:33]:
```suggestion
    public function updateIndexNames($from, $to)
```

File: updates/v2.0.1/rename_indexes.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    {
```
Comment (mjauvin) [inline updates/v2.0.1/rename_indexes.php:34]:
```suggestion
    {
        $table = $from;
```

**Review by mjauvin — CHANGES_REQUESTED** [review]
No need to pass argument twice.

File: updates/v2.0.1/rename_indexes.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        'locales',
+        'messages',
+    ];
+
+    public function up()
+    {
+        foreach (self::TABLES as $table) {
+            $from = 'rainlab_translate_' . $table;
+            $to   = 'winter_translate_' . $table;
+            $this->updateIndexNames($from, $to, $to);
```
Comment (mjauvin) [inline updates/v2.0.1/rename_indexes.php:20]:
```suggestion
            $this->updateIndexNames($from, $to);
```

**Comment by mjauvin** [comment]
Thanks!

**Comment by matteotrubini** [comment]
> Excume my previous modification requests, I had a brain cramp when I wrote this... ;)

Mee too when I choosen variables like a nursery rhyme  :hand_over_mouth:

**Pull request merged**